src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -514,6 +539,27 @@ internal NamedTypeSymbol UnderlyingTupleType
 
         internal bool HasFriendlyNames => _hasFriendlyNames;
 
+        public override ImmutableArray<ITypeSymbol> TupleElementTypes
```

Comment:
Consider making this an explicit implementation. We don't want to use this method within a compiler by accident.